src/DynamoCore/Logging/DynamoAnalyticsClient.cs

```diff
@@ -233,16 +232,8 @@ public void Start()
 
                 //If not ReportingAnalytics, then set the idle time as infinite so idle state is not recorded.
                 Service.StartUp(product, new UserInfo(Session.UserId), TimeSpan.FromMinutes(30));
-                TrackPreferenceInternal("ReportingAnalytics", "", ReportingAnalytics ? 1 : 0);
-
-                if (Configuration.DebugModes.IsEnabled("ADPAnalyticsTracker"))
-                {
-                    TrackPreferenceInternal("ReportingADPAnalytics", "", ReportingADPAnalytics ? 1 : 0);
-                }
-                else
-                {
-                    TrackPreferenceInternal("ReportingUsage", "", ReportingUsage ? 1 : 0);
-                }
+                TrackPreferenceInternal("ReportingGoogleAnalytics", "", ReportingAnalytics ? 1 : 0);
```

Comment:
I think this will be inconsistent with what we are tracking right now.. Making it hard to calculate certain the GA events.. But the impact seems small enough

Reply:
Are you suggesting to revert the event name "ReportingGoogleAnalytics" back to "ReportingUsage" ?

Reply:
@pinzart Just want to you to be aware in case you are not. We are tracking events under preference setting change, currently only include the `ReportingAnalytics` event and `ReportingUsage`. If we change the event name here, this will not be treated as same event as before so we will not be able to compare the data from same time last year etc. So unless it is a must do, I don't think we need to touch this only for code clarity, what do you think?

Reply:
I understand. Since we are obsoleting the Instrumentation logic (ReportingUsage) then we probably do not need to track that.
So Isuggest:
1. Remove the ReportUsage event (already done in PR).
2. Keep "ReportingAnalytics " event for compatibility reasons.
3. Add ADP and Google preferences so that they can individually be tracked
So basically :
`
TrackPreferenceInternal("ReportingAnalytics", "", ReportingAnalytics ? 1 : 0); //Either adp or google
TrackPreferenceInternal("ReportingADPAnalytics", "", ReportingADPAnalytics ? 1 : 0);//Adp preference
TrackPreferenceInternal("ReportingGoogleAnalytics", "", ReportingGoogleAnalytics ? 1 : 0);//Google preference
`
Or should we just completely revert to the old way (just reportingUsage + reportingAnalytics)?

Reply:
@pinzart By looking at all the history, given we are only tracking minimal set, I am fine to proceed with the current change and add more events tracking for preference setting later